metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraObjectsImporterThroughDBTest.java

```diff
@@ -29,12 +30,17 @@
 import uk.ac.ebi.ampt2d.metadata.importer.MetadataImporterMainApplication;
 import uk.ac.ebi.ampt2d.metadata.importer.ObjectsImporter;
 import uk.ac.ebi.ampt2d.metadata.persistence.entities.Analysis;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.Sample;
 import uk.ac.ebi.ampt2d.metadata.persistence.entities.Study;
 import uk.ac.ebi.ampt2d.metadata.persistence.repositories.AnalysisRepository;
 import uk.ac.ebi.ampt2d.metadata.persistence.repositories.ReferenceSequenceRepository;
 import uk.ac.ebi.ampt2d.metadata.persistence.repositories.StudyRepository;
 
 import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
```

Comment:
All the above four can be removed

Reply:
Fixed in https://github.com/EBIvariation/metadata-ws/pull/55/commits/0e5f9b50a3fc908dd654a8d64e7aac5919050f3a.